tests/plugins/imagebase/features/link.html

```diff
@@ -0,0 +1,7 @@
+<div id="upcastTest">
+	<figure>
+		<a href="http://foo">
+			<img src="http://foo">
```

Comment:
By using this fake src your tests are causing good amount errors in dev console. It can be simply fixed by using our common assets, e.g. `%BASE_PATH%_assets/logo.png`.